UndertaleModTool/Editors/UndertaleRoomEditor.xaml.cs

```diff
@@ -103,6 +103,23 @@ public void SaveImagePNG(Stream outfile)
             visualOffProp.SetValue(roomCanvas, prevOffset);
         }
 
+        public static void CheckAndRearrangeLayers(UndertaleRoom room)
```

Comment:
Documentation for the method would be nice.

Reply:
Is this fine?
```xml
/// <summary>
/// Checks if the layers are ordered by depth.<br/>
/// If they are not, then it will ask if they should be ordered.<br/>
/// If yes, then it will rearrange the layers.
/// </summary>
/// <param name="room">The <see cref="UndertaleRoom"/>, whose layers should be checked and rearranged.</param>
```